externs/shaka/manifest.js

```diff
@@ -198,6 +199,11 @@ shaka.extern.DrmInfo;
  *   The Variant's language, specified as a language code. <br>
  *   See {@link https://tools.ietf.org/html/rfc5646} <br>
  *   See {@link http://www.iso.org/iso/home/standards/language_codes.htm}
+ * @property {number} disabledUntilTime
+ *   <i>Defaults to 0.</i> <br>
+ *   0 means the variant is enabled. The Player will set this value to
+ *   "Date.now + config.streaming.maxDisabledTime" and once this maxDisabledTime
```

Comment:
Date.now returns milliseconds and config.streaming.maxDisabledTime returns seconds, so the comment is wrong.

Should be: `(Date.now() / 1000) + config.streaming.maxDisabledTime`